test/testApolloServer.js

```diff
@@ -185,6 +199,22 @@ describe('ApolloServer', () => {
       );
     });
   });
+
+  it('will append resolvers to a js schema', () => {
+    const app = express();
+    const jsServer = apolloServer({
+      schema: testJSSchema,
+      resolvers: testResolvers
+    });
+    app.use('/graphql', jsServer);
+    return request(app).get(
+      '/graphql?query={stuff}'
+    ).then((res) => {
+      const fields = testJSSchema.getType("RootQuery").getFields();
+      expect(fields['usecontext']['resolve']).to.exist;
```

Comment:
Great point.  I was trying to test for the very specific case but totally agree.